api/app/tests/weather_models/endpoints/test_models_predictions_most_recent_GDPS_[838]_response.json

```diff
@@ -12,7 +12,9 @@
                     "start_day": 15,
                     "end_month": 8,
                     "end_day": 31
-                }
+                },
+                "elevation": 821,
+                "wfwx_station_uuid": ""
```

Comment:
Does this actually comeback as "" from the server? Doesn't seem right? Could we not rather please get some real UUID's in here?